tests/legacy-cli/e2e/tests/basic/standalone.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * @fileoverview Tests the minimal conversion of a newly generated application
+ * to use a single standalone component.
+ */
+
+import { writeFile } from '../../utils/fs';
+import { ng } from '../../utils/process';
+
+/**
+ * An application main file that uses a standalone component with
+ * bootstrapApplication to start the application.
+ */
+const STANDALONE_MAIN_CONTENT = `
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { bootstrapApplication } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  template: \`
+    <ng-template [ngIf]="isVisible">
```

Comment:
Nit: I assume we're using `ng-template` here to validate that the `CommonModule` import worked, would be good to leave a comment stating that explicitly, otherwise it feels like an unnecessary implementation detail.